auto_joiner.py

```diff
@@ -617,7 +617,12 @@ def main():
 
     print("Waiting for correct page...", end='')
     if wait_until_found("#teams-app-bar", 60 * 5) is None:
-        exit(1)
+        # click the Try again button if teams load error
+        try_again = wait_until_found("button[class*='ts-btn ts-btn-primary oops-button']", 10)
```

Comment:
Can be shortened to "button.oops-button", please test if this finds the correct button (and not multiple elements)

Reply:
Yes the element can be shortened, and it finds the correct button and clicks it